build-scripts/copyDistFiles.js

```diff
@@ -0,0 +1,57 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+'use strict'
+
+//
+// Called from `npm run compile`.
+// Copies vended dependencies (SSM, ASL, …) to `dist`.
+//
+
+const fs = require('fs-extra')
+const path = require('path')
+
+const repoRoot = path.dirname(__dirname)
+
+async function copyFiles(name, files) {
+    try {
+        console.log(`copying artifacts for "${name}"...`)
+        for (let [from, to] of files) {
+            await fs.copy(path.resolve(repoRoot, from), path.resolve(repoRoot, to), {
+                overwrite: true,
+                errorOnExist: false,
+            })
+        }
+    } catch (e) {
+        console.error(`error: failed to copy artifacts for "${name}"`, e)
+        // Exit with non-zero exit code in order to fail the build if copying of any artifact fails.
+        process.exit(1)
+    }
+}
+
+;(async () => {
+    copyFiles('SSM Document Language Service', [
+        ['node_modules/aws-ssm-document-language-service/dist/server.js', 'dist/src/ssmDocument/ssm/ssmServer.js'],
+        [
+            'node_modules/aws-ssm-document-language-service/dist/server.js.LICENSE.txt',
+            'dist/src/ssmDocument/ssm/ssmServer.js.LICENSE.txt',
+        ],
+        [
+            'node_modules/aws-ssm-document-language-service/dist/server.js.map',
+            'dist/src/ssmDocument/ssm/ssmServer.js.map',
+        ],
+    ])
+
+    copyFiles('ASL server', [
+        ['node_modules/aws-ssm-document-language-service/dist/server.js', 'dist/src/stepFunctions/asl/aslServer.js'],
```

Comment:
This replaces the `onEnd` step that was done in `webpack.config.js`. @alanbo can you verify that ASL server is still working with this PR? 

You can get the artifact from this PR like so: https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#ci-artifact

Reply:
um just noticed a bad typo here. fixing..